### DISCUSSION TITLE
Allow Insecure TLS on *arr clients

### DISCUSSION THREAD

**Discussion by txtsd** [autobrr/autobrr#1792] [Ideas]
The qBittorrent client already has `Skip TLS verification (insecure)`.
Let's bring it to the other clients too so we can use https with our internal/local domains.

**Comment by mikelauskas** [comment]
I'm having the same problem, I think I'll take a look at the code to see if I can figure it out myself, but it would be great if one of the developers would take some interest in this.

**Comment by zze0s** [comment]
Done in https://github.com/autobrr/autobrr/pull/2335
